### PR TITLE
CB-4605 Freeipa password generator

### PR DESCRIPTION
**Pull request by tiborpopovics**

### CODE REVIEW

File: .cbd_history [hunk 1]
```diff
@@ -0,0 +1 @@
+2019-11-27 16:55:08 tiborpopovics kill
```
Comment (keyki) [inline .cbd_history]: this file is not needed

File: .cbd_output_history [hunk 2]
```diff
@@ -0,0 +1,7 @@
+2019-11-27 16:55:07 tiborpopovics kill
```
Comment (keyki) [inline .cbd_output_history]: same

File: .starttime [hunk 3]
```diff
@@ -0,0 +1 @@
+1574870107
```
Comment (keyki) [inline .starttime]: same

File: Profile [hunk 4]
```diff
@@ -0,0 +1,4 @@
+export CB_INSTANCE_UUID="24b26578-442c-4f4b-acfe-2a37da4ce769"
+export CB_INSTANCE_NODE_ID="c1cc1655-e6b9-4f00-a0b0-0876a9782492"
+export CB_INSTANCE_PROVIDER="unknown"
+export CB_INSTANCE_REGION="unknown"
```
Comment (keyki) [inline Profile]: same

File: common/src/main/java/com/sequenceiq/cloudbreak/util/FreeIpaPasswordUtil.java [hunk 5]
```diff
@@ -0,0 +1,31 @@
+package com.sequenceiq.cloudbreak.util;
+
+import org.apache.commons.lang3.RandomStringUtils;
+
+public class FreeIpaPasswordUtil {
+
+    private static final int PWD_PART_LENGTH = 8;
+
+    private static final String SPECIAL_CHARS = "?!./-*_+";
```
Comment (keyki) [inline common/src/main/java/com/sequenceiq/cloudbreak/util/FreeIpaPasswordUtil.java]: Can you please remove `/` so we can use the password safely in console

File: .cbd_history (hunk 1 above)
Comment (tiborpopovics) [reply .cbd_history]: deleted

File: .cbd_output_history (hunk 2 above)
Comment (tiborpopovics) [reply .cbd_output_history]: deleted

File: .starttime (hunk 3 above)
Comment (tiborpopovics) [reply .starttime]: deleted

File: Profile (hunk 4 above)
Comment (tiborpopovics) [reply Profile]: deleted

**Pull request merged**